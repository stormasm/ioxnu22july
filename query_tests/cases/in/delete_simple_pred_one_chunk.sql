-- Demonstrate soft deleted rows will not be return to queries
-- IOX_SETUP: OneDeleteSimpleExprOneChunk

-- select *
SELECT * from cpu;

SELECT time, bar from cpu;

SELECT min(bar), max(bar) from cpu;

SELECT time from cpu;

SELECT max(time)  from cpu;
SELECT min(time)  from cpu group by bar;
SELECT bar, min(time)  from cpu group by bar;

SELECT count(time), max(time)  from cpu;

SELECT count(time)  from cpu;

SELECT count(time), count(*), count(bar), min(bar), max(bar), min(time), max(time)  from cpu;

----------------------------------------------------------------
-- Now add selection predicate
SELECT * from cpu where bar = 2.0;

SELECT * from cpu where bar != 2.0;

SELECT count(time), count(*), count(bar), min(bar), max(bar), min(time), max(time)  from cpu where bar= 2.0;

SELECT count(time), count(*), count(bar), min(bar), max(bar), min(time), max(time)  from cpu where bar != 2.0;

SELECT time from cpu where bar=2;

SELECT bar from cpu where bar!= 2;


