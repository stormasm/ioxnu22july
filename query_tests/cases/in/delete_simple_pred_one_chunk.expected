-- Test Setup: OneDeleteSimpleExprOneChunk
-- SQL: SELECT * from cpu;
+-----+--------------------------------+
| bar | time                           |
+-----+--------------------------------+
| 2   | 1970-01-01T00:00:00.000000020Z |
+-----+--------------------------------+
-- SQL: SELECT time, bar from cpu;
+--------------------------------+-----+
| time                           | bar |
+--------------------------------+-----+
| 1970-01-01T00:00:00.000000020Z | 2   |
+--------------------------------+-----+
-- SQL: SELECT min(bar), max(bar) from cpu;
+--------------+--------------+
| MIN(cpu.bar) | MAX(cpu.bar) |
+--------------+--------------+
| 2            | 2            |
+--------------+--------------+
-- SQL: SELECT time from cpu;
+--------------------------------+
| time                           |
+--------------------------------+
| 1970-01-01T00:00:00.000000020Z |
+--------------------------------+
-- SQL: SELECT max(time)  from cpu;
+--------------------------------+
| MAX(cpu.time)                  |
+--------------------------------+
| 1970-01-01T00:00:00.000000020Z |
+--------------------------------+
-- SQL: SELECT min(time)  from cpu group by bar;
+--------------------------------+
| MIN(cpu.time)                  |
+--------------------------------+
| 1970-01-01T00:00:00.000000020Z |
+--------------------------------+
-- SQL: SELECT bar, min(time)  from cpu group by bar;
+-----+--------------------------------+
| bar | MIN(cpu.time)                  |
+-----+--------------------------------+
| 2   | 1970-01-01T00:00:00.000000020Z |
+-----+--------------------------------+
-- SQL: SELECT count(time), max(time)  from cpu;
+-----------------+--------------------------------+
| COUNT(cpu.time) | MAX(cpu.time)                  |
+-----------------+--------------------------------+
| 1               | 1970-01-01T00:00:00.000000020Z |
+-----------------+--------------------------------+
-- SQL: SELECT count(time)  from cpu;
+-----------------+
| COUNT(cpu.time) |
+-----------------+
| 1               |
+-----------------+
-- SQL: SELECT count(time), count(*), count(bar), min(bar), max(bar), min(time), max(time)  from cpu;
+-----------------+-----------------+----------------+--------------+--------------+--------------------------------+--------------------------------+
| COUNT(cpu.time) | COUNT(UInt8(1)) | COUNT(cpu.bar) | MIN(cpu.bar) | MAX(cpu.bar) | MIN(cpu.time)                  | MAX(cpu.time)                  |
+-----------------+-----------------+----------------+--------------+--------------+--------------------------------+--------------------------------+
| 1               | 1               | 1              | 2            | 2            | 1970-01-01T00:00:00.000000020Z | 1970-01-01T00:00:00.000000020Z |
+-----------------+-----------------+----------------+--------------+--------------+--------------------------------+--------------------------------+
-- SQL: SELECT * from cpu where bar = 2.0;
+-----+--------------------------------+
| bar | time                           |
+-----+--------------------------------+
| 2   | 1970-01-01T00:00:00.000000020Z |
+-----+--------------------------------+
-- SQL: SELECT * from cpu where bar != 2.0;
++
++
-- SQL: SELECT count(time), count(*), count(bar), min(bar), max(bar), min(time), max(time)  from cpu where bar= 2.0;
+-----------------+-----------------+----------------+--------------+--------------+--------------------------------+--------------------------------+
| COUNT(cpu.time) | COUNT(UInt8(1)) | COUNT(cpu.bar) | MIN(cpu.bar) | MAX(cpu.bar) | MIN(cpu.time)                  | MAX(cpu.time)                  |
+-----------------+-----------------+----------------+--------------+--------------+--------------------------------+--------------------------------+
| 1               | 1               | 1              | 2            | 2            | 1970-01-01T00:00:00.000000020Z | 1970-01-01T00:00:00.000000020Z |
+-----------------+-----------------+----------------+--------------+--------------+--------------------------------+--------------------------------+
-- SQL: SELECT count(time), count(*), count(bar), min(bar), max(bar), min(time), max(time)  from cpu where bar != 2.0;
+-----------------+-----------------+----------------+--------------+--------------+---------------+---------------+
| COUNT(cpu.time) | COUNT(UInt8(1)) | COUNT(cpu.bar) | MIN(cpu.bar) | MAX(cpu.bar) | MIN(cpu.time) | MAX(cpu.time) |
+-----------------+-----------------+----------------+--------------+--------------+---------------+---------------+
| 0               | 0               | 0              |              |              |               |               |
+-----------------+-----------------+----------------+--------------+--------------+---------------+---------------+
-- SQL: SELECT time from cpu where bar=2;
+--------------------------------+
| time                           |
+--------------------------------+
| 1970-01-01T00:00:00.000000020Z |
+--------------------------------+
-- SQL: SELECT bar from cpu where bar!= 2;
++
++
